src/api/app/mailers/event_mailer.rb

```diff
@@ -21,6 +21,9 @@ def event(subscribers, e)
     subscribers = subscribers.to_a
     return if subscribers.empty?
 
+    tos = subscribers.map(&:display_name).reject(&:blank?)
```

Comment:
Shouldn't we test this condition on `src/api/spec/mailers/event_mailer_spec.rb`

Reply:
What is `tos`?

Reply:
I'm working on specs for this change

Reply:
`tos` is the plural of `to` addresses :stuck_out_tongue:

Reply:
Ishhhh, could we have a better name for this?

Reply:
Yeah, I'll change it